convert/jenkinsjson/convert.go

```diff
@@ -801,12 +848,21 @@ func mergeRunSteps(steps *[]StepWithID) {
 			merged = append(merged, cursor)
 			cursor = current
 			pushed = true
+
+			if len(*steps) == i+1 {
+				fmt.Println("Inside if line 875")
+				isLastStep = true
+			}
 		}
 	}
 
 	if !pushed {
 		merged = append(merged, cursor)
 	}
+
+	if pushed && isLastStep {
```

Comment:
please revert this change and sync with latest. It should have been fixed already. If not please contact me with the example you are trying